test/WebSites/ActionResultsWebsite/Controllers/HomeController.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc;
+
+namespace ActionResultsWebSite
+{
+    public class HomeController : Controller
```

Comment:
Could we come up with an alternate name for this? `HomeController` is overused in our solution and makes looking these tests up particularly hard. `ActionResultsVerificationController` perhaps?